game/graphics/display.cpp

```diff
@@ -117,7 +118,7 @@ void KillDisplay(std::shared_ptr<GfxDisplay> display) {
 
   if (GetMainDisplay() == display) {
     // killing the main display, kill all children displays too!
-    for (int i = 1; i < g_displays.size(); ++i) {
+    for (size_t i = 1; i < g_displays.size(); ++i) {
       KillDisplay(g_displays.at(i));
     }
```

Comment:
This loop is actually totally wrong, my bad... This is only gonna kill every second sub-display.